tests/results/reconfig/v1.3.0/1.3.0-oss.md

```diff
@@ -0,0 +1,188 @@
+# Results
```

Comment:
Once this PR is fully ready to go, this results file can be deleted and the results from the old directory can be moved over (and `reconfig` directory deleted).

Reply:
I will re-run the test results then remove this when I push my PR.